changelog/9780.doc.md

```diff
@@ -0,0 +1 @@
+Updates quick install documentation with optional venv step, better pip install instructions, & M1 warning
```

Comment:
optional: add link to the "install documentation" in the changelog to make it easier discoverable for our users